omr/startup/omrvmstartup.cpp

```diff
@@ -325,7 +325,7 @@ OMR_Initialize_VM(OMR_VM **omrVMSlot, OMR_VMThread **omrVMThreadSlot, void *lang
 	OMR_VM *omrVM = NULL;
 	OMR_VMThread *vmThread = NULL;
 	OMRPORT_ACCESS_FROM_OMRPORT(&portLibrary);
-
+	
```

Comment:
Can we remove the tab here?

Reply:
That was an oversight. I will fix that as a part of any revisions I do in this PR.